src/python_routing_v02/troute/routing/diffusive_utils.py

```diff
@@ -454,14 +454,15 @@ def diffusive_input_data_v02(
     """
 
     # diffusive time steps info.
-    dt_ql_g = geo_data[0,0] * qts_subdivisions
-    dt_ub_g = geo_data[0,0] * qts_subdivisions # TODO: make this timestep the same as the simulation timestep
-    dt_db_g = geo_data[0,0] * qts_subdivisions # TODO: make this timestep the same as the simulation timestep
-    saveinterval_g = geo_data[0,0]
-    saveinterval_ev_g = geo_data[0,0]
-    dtini_g = geo_data[0,0]
+    dt = 300
```

Comment:
Need to clear this magic number before we merge. 
See https://github.com/NOAA-OWP/t-route/issues/365